addons/pos_self_order/static/src/app/services/data_service.js

```diff
@@ -46,4 +46,7 @@ patch(PosData.prototype, {
     async missingRecursive(recordMap) {
         return recordMap;
     },
+    async checkAndDeleteMissingOrders(results) {
+        return results;
+    },
```

Comment:
You don't have to return something, if you don't do anything, don't put anything.

```suggestion
    async checkAndDeleteMissingOrders(results) {},
```